_layouts/services.html

```diff
@@ -151,6 +154,28 @@
     opacity: 0.8;
     filter: contrast(0.8);
     mix-blend-mode: color-burn;
+    mask-image: linear-gradient(
+      to left,
+      rgba(0, 0, 0, 0.2) 0%,
+      rgba(0, 0, 0, 1) 100%
+    );
+    -webkit-mask-image: linear-gradient(
+      to left,
+      rgba(0, 0, 0, 0.2) 0%,
+      rgba(0, 0, 0, 1) 100%
+    );
+  }
+  .flex-row:not(.top):nth-last-child(even) img.side-image {
```

Comment:
What is `nth-last-child(even)` supposed to target here? Very complex CSS rule that makes this very hard to understand!

Consider adding a targeting CSS class instead of doing this complex deep targeting. Will make code easier to read and maintain.

Reply:
I agree that the `.flex-row:not(.top)` is not a great choice here, but the `nth-last-child(even)` is a simple css selector that targets every even child, this allows for all of the children to follow that same class structure in html and make their appearance depend on their order of rendering on the site

Reply:
Fixed in [bba235b](https://github.com/derangeddk/deranged.dk/pull/114/commits/bba235b1e04a963b9238637cf40a19040a3e8a74)